Bottom: 0798fe52e15039960c3d67a9143a90fb995e8632
Top:    1647bb2d69e06b5b0dd47de329909a9c1d995599
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-05-08 13:39:50 +0100

Fix crash hovering over unconnected element


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 4922cb4..346a6db 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1989,7 +1989,7 @@ describe_location (LocationType X, LocationType Y)
       GetLayerNumber (PCB->Data, (LayerTypePtr) ptr1) >= max_layer)
     return NULL;
 
-  elename = UNKNOWN (NAMEONPCB_NAME ((ElementTypePtr) ptr1));
+  elename = "DUMMY"; // UNKNOWN (NAMEONPCB_NAME ((ElementTypePtr) ptr1));
   pinname = ConnectionName (type, ptr1, ptr2);
 
   if (pinname == NULL)
